### PR TITLE
Hotfix - saving and retrieving authorization data (REST and RPC)

### PR DESCRIPTION
**Pull request by webimpress**
Fix for issue reported in #126 and #127

### CODE REVIEW

**Comment by jguittard** [comment]
Thanks @webimpress !

**Comment by weierophinney** [comment]
@jguittard Please do the following as well:

- Merge to develop! (see the [maintainers guide to merging](https://github.com/zendframework/maintainers/blob/master/MAINTAINERS.md#workflow-for-merging-pull-requests) for details)
- In this particular repo, we also need to rebuild the distribution when merging, to ensure it's up-to-date for those using dev-master/dev-develop.

**Comment by jguittard** [comment]
Will do, thx for the reminder!

File: src/apigility-ui/service/api.service.js [hunk 1]
```diff
@@ -448,11 +450,13 @@
     };
 
     this.getAuthorizationRpc = function(module, version, name, callback) {
-      var moduleParam = this.normalizeModuleName(module);
+      var moduleParam = this.normalizeModuleName(module),
+          controllerName = name.replace(/-/g, '\\'),
```
Comment (weierophinney) [inline src/apigility-ui/service/api.service.js]: The backend understands both `-` and `\\` as separation characters within the controller service names, so this normalization isn't strictly necessary.

**Pull request merged**